Source/Core/DolphinQt2/WiiUpdate.cpp

```diff
@@ -74,46 +119,25 @@ void PerformOnlineUpdate(const std::string& region, QWidget* parent)
               true);
           return !was_cancelled.IsSet();
         },
-        region);
+        std::forward<Args>(args)...);
     Core::QueueHostJob([&dialog] { dialog.close(); }, true);
     return res;
   });
 
   dialog.exec();
+  Core::HostDispatchJobs();
```

Comment:
Hmm. When `Core::QueueHostJob` is called multiple times, are those jobs guaranteed to be run in-order? In that case, won't all the jobs referencing the dialog be completed by the time `dialog.exec()` returns?